cachelib/allocator/memory/MemoryAllocatorStats.h

```diff
@@ -54,6 +54,17 @@ struct ACStats {
   constexpr size_t getTotalFreeMemory() const noexcept {
     return Slab::kSize * freeSlabs + freeAllocs * allocSize;
   }
+
+  constexpr double usageFraction() const noexcept {
+    if (usedSlabs == 0)
+      return 0.0;
+
+    return activeAllocs / (usedSlabs * allocsPerSlab);
+  }
+
+  constexpr size_t totalAllocatedSize() const noexcept {
+    return activeAllocs * allocSize;
+  }
```

Comment:
Are these going to be used inside cachelib by background workers? Otherwise can you move them into cachebench?

Reply:
We wanted to use the in BG workers initially but most likely we will use another mechanism with lower overhead. Done.